src/index.html

```diff
@@ -257,7 +257,91 @@ <h2>QUICK ID</h2>
 
         <section>
           <h2>CONTACT ME</h2>
-          <!-- Form Related Content for Lesson 2 -->
+        </section>
+      </div> -->
+
+      <div id="info" class="info-container">
+        <section id="info-quick-id" class="info-section">
```

Comment:
do you really need to have all these tag with Id or class attributes?
Id is not idea for attach styles since it can be hard to override, and class should define some re-useable pieces of styling.

Adding all the unnecessary selectors will increase your file size and can be tricky to maintain later on if someone else pick up your code.